### PR TITLE
Generalising article to include IOTA

### PR DESCRIPTION
**Pull request by Phyloiota**
Included IOTA and removed Shimmer-specific wording, some grammar and spelling updates

# Description of change

Article before was only focussed on Shimmer, updated to be valid for both networks

## Links to any relevant issues

Be sure to reference any related issues by adding `fixes #issue_number`.

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Documentation Enhancement

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [x ] I have followed the [contribution guidelines](https://github.com/iota-wiki/iota-wiki/blob/main/.github/CONTRIBUTING.md) for this project
- [ x] I have performed a self-review of my own changes
- [x ] I have made sure that added/changed links still work
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: docs/learn/governance/governance-tech.md [hunk 1]
```diff
@@ -1,36 +1,37 @@
...
   - Voting
   - Shimmer
+  - IOTA
   - Governance
 ---
 
-On-chain governance is essential for any decentralized DLT. The ability to make decisions through voting functionalities executed on the Blockchain / DAG removes a centralization point and gives token holders the power to make their own decisions. With Shimmer's Firefly Wallet, Shimmer token holders can now participate in the governance of the network.
-With the support of the IOTA Foundation, the Shimmer community has developed a highly flexible and secure system for casting votes directly on the Shimmer Tangle
-through the Firefly Wallet and counting the votes in the network’s Hornet nodes.
+On-chain governance is essential for any decentralized DLT. The ability to make decisions through voting functionalities executed on the Blockchain / DAG removes a centralization point and gives token holders the power to make their own decisions. IOTA and Shimmer token holders can now carry out governance decisions with the Firefly Wallet.
```
Comment (lucas-tortora) [inline docs/learn/governance/governance-tech.md:15]: "Carry out" in this context seems to imply that they execute the voted action instead of voting.

Comment (Phyloiota) [inline docs/learn/governance/governance-tech.md:15]:
```suggestion
On-chain governance is essential for any decentralized DLT. The ability to make decisions through voting functionalities executed on the Blockchain / DAG removes a centralization point and gives token holders the power to make their own decisions. IOTA and Shimmer token holders can now engage in governance with the Firefly Wallet.
```

Comment (Phyloiota) [reply docs/learn/governance/governance-tech.md:15]: Ah yes - that may be confusing, changed that

File: docs/learn/sidebars.ts [hunk 2]
```diff
@@ -141,7 +141,7 @@ module.exports = {
       label: 'Governance',
       collapsed: true,
       items: [
-        'governance/shimmer-governance-intro',
+        'governance/governance-intro',
```
Comment (Dr-Electron) [inline docs/learn/sidebars.ts:144]: Let's undo this, it got already fixed in your other PR

```suggestion
        'governance/shimmer-governance-intro',
```

**Review by Dr-Electron — APPROVED** [review]

**Pull request merged**